### ISSUE TITLE
3DView - dots hide each other

### ISSUE THREAD

**Issue by BillPascoe** [HughCraig/GHAP#343]
If there's 2 dots in the same place in 3D View, you can't click on the dot and navigate through all the dots like in some other Views. One dot simply hides the other dot. This might depend on the ArcGIS JS API and so be tricky. Worth seeing if we can do something about this.

**Comment by HughCraig** [comment]
Yes, this is an important one, especially because there is no indication in 3D Views that there is anything under the top dot. The cluster view handles this perfectly -- it indicates through the number appearing in the map dot how many places there are and then allows you to 'browse features' to inspect the details for each place.
